ui/user/src/lib/components/admin/GuardrailForm.svelte

```diff
@@ -0,0 +1,204 @@
+	class="flex h-full w-full flex-col gap-4"
+	out:fly={{ x: 100, duration }}
+	in:fly={{ x: 100, delay: duration }}
+>
+	<div class="flex grow flex-col gap-4" out:fly={{ x: -100, duration }} in:fly={{ x: -100 }}>
+		{#if topContent}
+			{@render topContent()}
+		{/if}
+		<h1 class="text-2xl font-semibold">Create Guardrail</h1>
+
+		<div
+			class="dark:bg-surface2 dark:border-surface3 rounded-lg border border-transparent bg-white p-4"
+		>
+			<div class="flex flex-col gap-6">
+				<div class="flex flex-col gap-2">
+					<label for="mcp-catalog-name" class="flex-1 text-sm font-light capitalize"> Name </label>
+					<input
+						id="mcp-catalog-name"
+						bind:value={guardrail.displayName}
+						class="text-input-filled mt-0.5"
+					/>
+				</div>
+			</div>
+		</div>
+
+		<div class="flex flex-col gap-2">
+			<div class="mb-2 flex items-center justify-between">
+				<h2 class="text-lg font-semibold">Webhook URLs</h2>
+				<div class="relative flex items-center gap-4">
+					<button
+						class="button-primary flex items-center gap-1 text-sm"
+						onclick={() => {
+							addUrlDialog?.showModal();
+						}}
+					>
+						<Plus class="size-4" /> Add URL
+					</button>
+				</div>
+			</div>
+
+			<Table data={urlsTableData} fields={['url']} noDataMessage="No URLs added.">
+				{#snippet actions(d)}
+					<button
+						class="icon-button hover:text-red-500"
+						onclick={() => {
+							guardrail.urls = guardrail.urls?.filter((url) => url !== d.id) ?? [];
+						}}
+						use:tooltip={'Remove URL'}
+					>
+						<Trash2 class="size-4" />
+					</button>
+				{/snippet}
+			</Table>
+		</div>
+
+		<div class="flex flex-col gap-2">
+			<div class="mb-2 flex items-center justify-between">
+				<h2 class="text-lg font-semibold">Models</h2>
+				<div class="relative flex items-center gap-4">
+					<button
+						class="button-primary flex items-center gap-1 text-sm"
+						onclick={() => {
+							addModelDialog?.showModal();
+						}}
+					>
+						<Plus class="size-4" /> Add Model
+					</button>
+				</div>
+			</div>
+			<Table data={guardrail.models} fields={['name']} noDataMessage="No models added.">
+				{#snippet actions(d)}
+					<button
+						class="icon-button hover:text-red-500"
+						onclick={() => {
+							guardrail.models = guardrail.models?.filter((model) => model.id !== d.id) ?? [];
+						}}
+						use:tooltip={'Remove Model'}
+					>
+						<Trash2 class="size-4" />
+					</button>
+				{/snippet}
+			</Table>
+		</div>
+	</div>
+	<div
+		class="bg-surface1 sticky bottom-0 left-0 flex w-full justify-end gap-2 py-4 text-gray-400 dark:bg-black dark:text-gray-600"
+		out:fly={{ x: -100, duration }}
+		in:fly={{ x: -100 }}
+	>
+		<div class="flex w-full justify-end gap-2">
+			<button
+				class="button text-sm"
+				onclick={() => {
+					goto('/v2/admin/guardrails');
+				}}
+			>
+				Cancel
+			</button>
+			<button
+				class="button-primary text-sm disabled:opacity-75"
+				onclick={async () => {
+					// TODO:
```

Comment:
Anything that needs to be done here?

Reply:
No, this doesn't have any functionality right now.